DESCRIPTION

```diff
@@ -2,13 +2,18 @@ Type: Package
 Package: epidatasets
 Title: Epidemiological Data for Delphi Tooling Examples
 Version: 0.0.1
-Authors@R: 
-    person(c("Daniel", "J."), "McDonald", , "daniel@stat.ubc.ca", role = c("cre", "aut"))
+Authors@R: c(
+    person(c("Daniel", "J."), "McDonald", email="daniel@stat.ubc.ca", role = c("cre", "aut")),
+    person("Johns Hopkins University Center for Systems Science and Engineering", role = "dtc", comment = "Owner of COVID-19 cases and deaths data sourced from the COVID-19 Data Repository"),
+    person("Johns Hopkins University", role = "cph", comment = "Copyright holder of COVID-19 cases and deaths data sourced from the COVID-19 Data Repository"),
+    person("Carnegie Mellon University Delphi Group", role = "dtc", comment = "Owner of masking and social-distancing data sourced from the COVID-19 Trends and Impacts Survey"),
+    person("Carnegie Mellon University", role = "cph", comment = "Copyright holder of masking and social-distancing data sourced from the COVID-19 Trends and Impacts Survey")
```

Comment:
question: Not sure this is true. Is Delphi the copyright holder?

Reply:
(Not quite an answer.  I based this on https://cmu-delphi.github.io/delphi-epidata/api/covidcast_licensing.html:

> [...]
> - [COVID-19 Trends and Impact Survey](https://cmu-delphi.github.io/delphi-epidata/api/covidcast-signals/fb-survey.html)
> 
> You may use this data for any purpose, provided you attribute us as the original source, as shown above.

So if this is imprecise, we also need to update there.)

Reply:
Dropping the CMU CTIS `cph` line. We can resolve it as part of issue cmu-delphi/epiprocess#348